code/datums/gamemode/factions/syndicate/nukeops.dm

```diff
@@ -184,26 +165,91 @@
+			synd_mob.equip_to_slot_or_drop(new /obj/item/weapon/storage/pill_bottle/inaprovaline(synd_mob), slot_in_backpack)
+			synd_mob.put_in_hand(GRASP_RIGHT_HAND, new /obj/item/roller/surgery(synd_mob))
+		if("Engineering") //Mister deconstruction, C4 and efficient. Engineers have shotguns because stereotype, and eswords for utility
+			synd_mob.equip_to_slot_or_drop(new /obj/item/clothing/glasses/scanner/meson/prescription(synd_mob), slot_glasses)
+			synd_mob.equip_to_slot_or_drop(new /obj/item/weapon/gun/projectile/shotgun/pump/combat(synd_mob), slot_s_store) //Only place we can store it, it will drop on the ground for plasmamen
+			synd_mob.equip_to_slot_or_drop(new /obj/item/weapon/storage/belt/utility/complete(synd_mob), slot_belt)
+			synd_mob.equip_to_slot_or_drop(new /obj/item/weapon/storage/box/lethalshells(synd_mob), slot_in_backpack)
+			synd_mob.equip_to_slot_or_drop(new /obj/item/weapon/melee/energy/sword(synd_mob), slot_in_backpack)
+			synd_mob.equip_to_slot_or_drop(new /obj/item/weapon/plastique(synd_mob), slot_in_backpack)
+			synd_mob.equip_to_slot_or_drop(new /obj/item/weapon/plastique(synd_mob), slot_in_backpack)
+			synd_mob.equip_to_slot_or_drop(new /obj/item/weapon/plastique(synd_mob), slot_in_backpack)
+			synd_mob.equip_to_slot_or_drop(new /obj/item/clothing/glasses/welding/superior(synd_mob), slot_l_store)
+			synd_mob.put_in_hand(GRASP_RIGHT_HAND, new /obj/item/clothing/shoes/magboots/syndie/elite(synd_mob))
+		if("Stealth") //WE STELT. Has an energy crossbow primary and a silenced pistol with magazines, along with a basic kit of infiltration items you could need to not nuke the Ops' credits
+			synd_mob.equip_to_slot_or_drop(new /obj/item/clothing/glasses/thermal/syndi(synd_mob), slot_glasses)
+			synd_mob.equip_to_slot_or_drop(new /obj/item/clothing/mask/gas/voice(synd_mob), slot_wear_mask)
+			synd_mob.equip_to_slot_or_drop(new /obj/item/weapon/gun/projectile/silenced(synd_mob), slot_belt)
+			synd_mob.equip_to_slot_or_drop(new /obj/item/ammo_storage/magazine/c45(synd_mob), slot_in_backpack)
+			synd_mob.equip_to_slot_or_drop(new /obj/item/weapon/card/emag(synd_mob), slot_in_backpack)
+			synd_mob.equip_to_slot_or_drop(new /obj/item/weapon/pen/paralysis(synd_mob), slot_in_backpack)
+			synd_mob.equip_to_slot_or_drop(new /obj/item/weapon/gun/energy/crossbow(synd_mob), slot_l_store)
+		if("Ship and Cameras") //The guy who stays on the shuttle and goes braindead. This kit is basically useless outside of giving you the coveted teleporter board, saving your team 40 points if you use it
+			synd_mob.equip_to_slot_or_drop(new /obj/item/clothing/glasses/thermal/syndi(synd_mob), slot_glasses)
+			synd_mob.equip_to_slot_or_drop(new /obj/item/device/encryptionkey/binary(synd_mob), slot_in_backpack)
+			synd_mob.equip_to_slot_or_drop(new /obj/item/device/megaphone/madscientist(synd_mob), slot_in_backpack)
+			synd_mob.equip_to_slot_or_drop(new /obj/item/weapon/circuitboard/teleporter(synd_mob), slot_l_store)
+
```

Comment:
Should probably make a helper method that runs a for-loop `n` amount of times and calls the specific `synd_mob.equip_to_slot_or_drop` with the specific argument.

Reply:
Not sure if I get the trick there, since that's how equipping mobs is done everywhere else

Reply:
So instead of calling this:

`synd_mob.equip_to_slot_or_drop(new /obj/item/weapon/plastique(synd_mob), slot_in_backpack)
synd_mob.equip_to_slot_or_drop(new /obj/item/weapon/plastique(synd_mob), slot_in_backpack)
synd_mob.equip_to_slot_or_drop(new /obj/item/weapon/plastique(synd_mob), slot_in_backpack)`

you will call

`equip_to_slot_or_drop_helper(3, /obj/item/weapon/plastique, synd_mob, slot_backpack)`

and the method will be something like

```
/mob/proc/equip_to_slot_or_drop_helper(var/times, var/obj/item/item, var/mob/living/carbon/human/synd_mob, var/slot)
	for (var/i = 0; i < times; i++)
		synd_mob.equip_to_slot_or_drop(new item(synd_mob), slot)